### PR TITLE
Create test for ncio

### LINKED ISSUE #11: Add testing

### PR DESCRIPTION
**Pull request by kgerheiser**
I took the test program here and added it to CMake:

https://github.com/NOAA-EMC/NCEPLIBS-ncio/issues/10#issuecomment-776119479

Just needs the data file to run.

Fixes #11

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -29,6 +29,11 @@ find_package(NetCDF REQUIRED Fortran)
 
 add_subdirectory(src)
 
+include(CTest)
+if(BUILD_TESTING)
+    add_subdirectory(tests)
+endif()
```
Comment (aerorahul) [inline CMakeLists.txt:35]: why is this in a if-block? It should be built regardless.  Running is optional.

Comment (kgerheiser) [reply CMakeLists.txt:35]: That's the variable CTest uses. It's on by default and is what's used in their documentation.

https://cmake.org/cmake/help/latest/module/CTest.html

**Comment by jswhit** [comment]
Thanks @kgerheiser!  After #16 is merged, `use module_fv3gfs_ncio` will need to be changed to `use module_ncio` in the test.  Where should the test data file be kept?  Its like 33mb, but I could create a lower res version that is maybe 1/4 the size.

**Comment by edwardhartnett** [comment]
What I prefer with netCDF testing is not to use data files, but to create them in testing.

In other words, make part of the test create a data file, and the rest of the test opens and checks it. In that way, we don't have to download any data files to our testing.

However, if we do need data files, we need to send a tarball to Kate which she will put on the ftp site for us...

**Pull request merged**